src/storage/src/table/mem_table.rs

```diff
@@ -0,0 +1,69 @@
+// Copyright 2022 Singularity Data
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+// http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+#![allow(dead_code)]
+#![allow(unused)]
```

Comment:
Remember to remove these lines in the next PR. 😁

Reply:
Ok, adding these two lines is to pass CI, and it will be removed when the interface is implemented.

Reply:
I think we should never allow unused. We can allow dead_code.